react/demo/app/samples/pagination/Custom.route.tsx

```diff
@@ -0,0 +1,84 @@
+import {Pagination as PaginationAgnosUI, WidgetsDefaultConfig} from '@agnos-ui/react';
```

Comment:
Why rename the ```Pagination``` to ```PaginationAgnosUI``` ? We never do this for all the other widgets, so this is currently not consistent with our other examples.
If you think it's better, why not, but we should stay consistent between widgets I think.

I would propose to keep Pagination for the agnosui component and export the route as CustomPagination or smth similar.

Reply:
ok

Reply:
I opened an issue